### PR TITLE
troubleshooting splunk cloud

### PR DESCRIPTION
**Pull request by jbabazadeh**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/XSUP-33231, https://jira-dc.paloaltonetworks.com/browse/XSUP-33338

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by ilappe — APPROVED** [review]
I think it's better to write the general error message 
something like `Connection to <name>.splunkcloud.com timed out`
instead of the entire log error

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 1]
```diff
@@ -1132,3 +1132,16 @@ The default port is 8088.
 ## Troubleshooting
 
 In case you encounter HTTP errors (e.g., IncompleteRead), we recommend using Python requests handler.
+
+If you encounter connectivity issues while using **Splunk Cloud** within **XSOAR8 or XSIAM** and receive the following error
```
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:1136]:
```suggestion
If you encounter connectivity issues while using **Splunk Cloud** within Cortex XSOAR8 or Cortex XSIAM you may receive the following error:
```

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 2]
```diff
@@ -1132,3 +1132,16 @@ The default port is 8088.
...
+
+    requests.exceptions.ConnectTimeout:
+    HTTPSConnectionPool(host='<name>.splunkcloud.com', port=8089)
+    : Max retries exceeded with url: /services/auth/login (Caused by ConnectTimeoutError(<urllib3.connection.HTTPSConnection object at 0x7fc389a4e170>,
+     'Connection to <name>.splunkcloud.com timed out. 
+    (connect timeout=None)'))
+
+To resolve this issue, add the IP addresses of XSOAR8 or XSIAM to the Splunk Cloud whitelist.
```
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:1144]:
```suggestion
To resolve this issue, add the IP addresses of Cortex XSOAR8 or Cortex XSIAM to the Splunk Cloud whitelist.
```

File: Packs/SplunkPy/Integrations/SplunkPy/README.md [hunk 3]
```diff
@@ -1132,3 +1132,16 @@ The default port is 8088.
...
+You can find the relevant IP addresses at:
+[Cortex XSOAR Administrator Guide](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSOAR/8/Cortex-XSOAR-Administrator-Guide/Enable-Access-to-Cortex-XSOAR) 
+under **Used for communication between Cortex XSOAR and customer resources** . Choose the IP address corresponding to your Cortex XSOAR region.
```
Comment (ShirleyDenkberg) [inline Packs/SplunkPy/Integrations/SplunkPy/README.md:1147]:
```suggestion
under **Used for communication between Cortex XSOAR and customer resources**. Choose the IP address corresponding to your Cortex XSOAR region.
```
What about for XSIAM?

**Comment by ShirleyDenkberg** [comment]
@ilappe Doc review completed.

**Pull request merged**